src/components/signup.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+
+class Signup extends Component {
```

Comment:
Please use functional components instead of class components

Reply:
Changes made